commands/Skill/bugjuice.js

```diff
@@ -0,0 +1,14 @@
+module.exports.execute = async (client, message) => {
+  return await message.channel.send(
+    Math.random() < 0.5 ? ':cockroach:' : ':cocktail:'
+  );
+};
+
+module.exports.config = {
+  name: 'bug juice',
```

Comment:
Keep commands one word in length. The first word of a command is parsed as the command, the rest as arguments.